test/node.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+var assert = require('simple-assert');
+var type = require('..');
+(function () {
+  if (typeof process === 'undefined' || typeof process.release !== 'object' || !process.release.name) {
+    return describe.skip;
+  }
+  return describe;
+}())('Node Specific', function () {
+
+
```

Comment:
Minor beauty thing in case you want to change the above: there is a double linebreak here, in the tests above there is always just one.